packages/drivers/file-driver/tsconfig.json

```diff
@@ -7,9 +7,10 @@
     "compilerOptions": {
         "rootDir": "./src",
         "outDir": "./dist",
-        "types": [ "node" ]
+        "types": [ "node" ],
+        "noUnusedLocals": false
```

Comment:
This is needed because IDocumentStorageServicePolicies and ISummaryContext are not used, but if they're not included, TypeScript will generate a dynamic import in the d.ts file.